api-reference/10 UI Components/dxFileManager/1 Configuration/onItemCopying.md

```diff
@@ -5,34 +5,173 @@ default: null
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+A function that is executed before a file or folder is copied.
 
 ##### param(e): Object
-<!-- Description goes here -->
+Information about the event.
 
 ##### field(e.cancel): Boolean | Promise<void>
-<!-- Description goes here -->
+Allows you to cancel the file or folder copy process.
 
 ##### field(e.component): dxFileManager
-<!-- Description goes here -->
+The UI component's instance.
 
 ##### field(e.destinationDirectory): FileSystemItem
-<!-- Description goes here -->
+The directory where the file or folder is copied to.
 
 ##### field(e.element): DxElement
-<!-- Description goes here -->
+#include common-ref-elementparam with { element: "component" }
 
 ##### field(e.errorCode): Number
-<!-- Description goes here -->
+#include filemanager-error-codes
 
 ##### field(e.errorText): String
-<!-- Description goes here -->
+The error message.
 
 ##### field(e.item): FileSystemItem
-<!-- Description goes here -->
+The copied file or folder.
```

Comment:
Maybe be better "The file or folder to be copied" since this event occurrs before copy operation. The same question about all other similar fields in the "-ing" events